tests/models/grounding_dino/test_modeling_grounding_dino.py

```diff
@@ -115,7 +150,9 @@ def prepare_config_and_inputs(self):
         pixel_values = floats_tensor([self.batch_size, self.num_channels, self.image_size, self.image_size])
         pixel_mask = torch.ones([self.batch_size, self.image_size, self.image_size], device=torch_device)
 
-        input_ids = ids_tensor([self.batch_size, self.max_text_len], self.num_labels)
+        # To avoid erros when running tests with `labels` `input_ids` have to follow this structure
+        input_ids = torch.tensor([101, 3869, 1012, 11420, 1012, 1012, 102])
```

Comment:
```suggestion
        # To avoid errors when running tests with `labels` `input_ids` have to follow this structure.
        # Start with 101 which means [SOS], 102 means [EOS], and 1012 means dot symbol for the seperation. (e.g. '[CLS] fish. shark. [SEP]')
        input_ids = torch.tensor([101, 3869, 1012, 11420, 1012, 102])
```
I think there is slight typo in the 1012 token is written twice. @EduardoPach